.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+
+os:
+  - linux
+  - osx
+
+node_js:
+  - '0.10'
+  - '0.12'
+  - '4'
+  - '5'
+  - 'node'
+  - 'iojs'
```

Comment:
This would be good, though a couple of suggestions:
* remove 'iojs'
* add node 6 and 7
* add SKIP_DTRACE=1 environment variable